src/openrct2/platform/linux.c

```diff
@@ -56,6 +56,8 @@ void platform_get_exe_path(utf8 *outPath, size_t outSize)
         log_fatal("failed to get process path");
 
     }
+#elif defined(__OpenBSD__)
+    strlcpy(exePath, "/usr/local/bin/", sizeof(exePath));
```

Comment:
Would a fallback mechanism be possible? Something along the lines "check if there is `/usr/local/bin/openrct2` executable" and perhaps "am I the `/usr/local/bin/openrct2`" and if these return false, warn user about it?